src/main/java/com/mitchej123/hodgepodge/mixins/late/extrautilities/MixinTileMiniChest.java

```diff
@@ -0,0 +1,27 @@
+package com.mitchej123.hodgepodge.mixins.late.extrautilities;
+
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.World;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import com.rwtema.extrautils.tileentity.chests.TileMiniChest;
+
+@Mixin(TileMiniChest.class)
```

Comment:
You should be able to just use one mixin which targets both classes: `@Mixin({TileFullChest.class, TileMiniChest.class})`